doc/api/cluster.md

```diff
@@ -649,6 +649,14 @@ finished.
 
 This can only be called from the master process.
 
+```js
+  const cluster = require('cluster');
+
+  if(cluster.IsMaster){
```

Comment:
```suggestion
  if (cluster.isMaster) {
```

Reply:
Thanks for your feedback, I'm gonna update the PR.